includes/class-llms-loader.php

```diff
@@ -230,11 +230,15 @@ public function autoload( $class ) {
 	 * @since 5.6.0 Include `LLMS_Prevent_Concurrent_Logins`.
 	 * @since 6.0.0 Included `LLMS_Block_Library`, `LLMS_Controller_Awards`, and `LLMS_Engagement_Handler`.
 	 *              Removed loading of class files that don't instantiate their class in favor of autoloading.
+	 * @since [version] Instantiated `LLMS_Shortcodes` before `LLMS_Controller_Orders`.
 	 *
 	 * @return void
 	 */
 	public function includes() {
 
+		// Instantiate LLMS_Shortcodes before LLMS_Controller_Orders.
+		new LLMS_Shortcodes();
```

Comment:
I know why this is done this way but instantiating a class in a method that only includes files seems like a break in an established pattern. I know the autoloader loads it and it performs the same function but I don't like it. I'd prefer to instantiate elsewhere instead of a method with an explicit purpose of "including files"